anaconda.sh

```diff
@@ -73,57 +73,51 @@ GUEST_IP="192.168.100.50"
 FIRMWARE=${FIRMWARE:-"bios"}
 PARTITION=${PARTITION:-"standard"}
 
+greenprint "Get tier-1 image info"
+REDHAT_ID=$(skopeo inspect --tls-verify=false "docker://${TIER1_IMAGE_URL}" | jq -r '.Labels."redhat.id"')
+REDHAT_VERSION_ID=$(skopeo inspect --tls-verify=false "docker://${TIER1_IMAGE_URL}" | jq -r '.Labels."redhat.version-id"')
+CURRENT_COMPOSE_ID=$(skopeo inspect --tls-verify=false "docker://${TIER1_IMAGE_URL}" | jq -r '.Labels."redhat.compose-id"')
+
 greenprint "Login quay.io"
 podman login -u "${QUAY_USERNAME}" -p "${QUAY_PASSWORD}" quay.io
 
-case "$TEST_OS" in
-    "rhel-9-4")
-        IMAGE_NAME="rhel9-rhel_bootc"
-        TIER1_IMAGE_URL="${RHEL_REGISTRY_URL}/${IMAGE_NAME}:rhel-9.4"
-        TIER1_IMAGE_URL="${IMAGE_URL-$TIER1_IMAGE_URL}"
-        CURRENT_COMPOSE_RHEL94=$(skopeo inspect --tls-verify=false "docker://${TIER1_IMAGE_URL}" | jq -r '.Labels."redhat.compose-id"')
-        sed "s/REPLACE_ME/${DOWNLOAD_NODE}/; s/REPLACE_COMPOSE_ID/${CURRENT_COMPOSE_RHEL94}/" files/rhel-9-4.template | tee rhel-9-4.repo > /dev/null
-        # sed "s/REPLACE_ME/${DOWNLOAD_NODE}/; s/REPLACE_COMPOSE_ID/latest-RHEL-9.4.0/" files/rhel-9-4.template | tee rhel-9-4.repo > /dev/null
-        ADD_REPO="COPY rhel-9-4.repo /etc/yum.repos.d/rhel-9-4.repo"
+case "$REDHAT_ID" in
+    "rhel")
+        # work with old TEST_OS variable value rhel-9-x
+        TEST_OS=$(echo "${REDHAT_ID}-${REDHAT_VERSION_ID}" | sed 's/\./-/')
+        sed "s/REPLACE_ME/${DOWNLOAD_NODE}/; s/REPLACE_COMPOSE_ID/${CURRENT_COMPOSE_ID}/" files/rhel-9-y.template | tee rhel-9-y.repo > /dev/null
+        ADD_REPO="COPY rhel-9-y.repo /etc/yum.repos.d/rhel-9-y.repo"
         ADD_RHC="RUN dnf install -y rhc"
-        # The current image is built based on RHEL-9.4.0-20240130.10. It doesn't include patched anaconda. Let's use latest as workaround
-        BOOT_LOCATION="http://${DOWNLOAD_NODE}/rhel-9/composes/RHEL-9/${CURRENT_COMPOSE_RHEL94}/compose/BaseOS/${ARCH}/os/"
-        # BOOT_LOCATION="http://${DOWNLOAD_NODE}/rhel-9/nightly/RHEL-9/latest-RHEL-9.4.0/compose/BaseOS/\$basearch/os/"
+        BOOT_LOCATION="http://${DOWNLOAD_NODE}/rhel-9/composes/RHEL-9/${CURRENT_COMPOSE_ID}/compose/BaseOS/${ARCH}/os/"
         OS_VARIANT="rhel9-unknown"
         BOOT_ARGS="uefi"
         CUT_DIRS=8
         ;;
-    "centos-stream-9")
-        IMAGE_NAME=${IMAGE_NAME:-"centos-bootc"}
-        TIER1_IMAGE_URL="quay.io/centos-bootc/${IMAGE_NAME}:stream9"
-        TIER1_IMAGE_URL="${IMAGE_URL-$TIER1_IMAGE_URL}"
```

Comment:
Could you help to change the `IMAGE_URL` to `TIER1_IMAGE_URL` in `.tekton/testing-farm.yaml`, I think this is only only thing left for this change.